docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
+
+Create a plugin
+
+**Example request**:
+
+To create a plugin named `plugin`
+
+```
+POST /plugins/plugin:latest/create HTTP/1.1
+
+Header should be set to "Content-Type: application/tar"
```

Comment:
Yes, they probably work both. If it doesn't make a difference to us, then I prefer `x-tar` to be consistent